HUD/GetScaleformMovieCursorSelection.md

```diff
@@ -0,0 +1,34 @@
+---
+ns: HUD
+aliases: ["0x632B2940C67F4EA9"]
+---
+## _GET_SCALEFORM_MOVIE_CURSOR_SELECTION
+ 
+```c
+// 0x632B2940C67F4EA9
+BOOL _GET_SCALEFORM_MOVIE_CURSOR_SELECTION(int scaleformHandle, bool* received, int* selectionType, int* slotIndex, int* p4);
```

Comment:
When changing parameter types, please add `cs_type(Any*)` before the new type to remain backwards compatible when using C#.

Reply:
should be alright in the new commit